app/routes/events/components/EventAdministrate/RegistrationTables.tsx

```diff
@@ -271,6 +282,20 @@ export const RegisteredTable = ({
           <PresenceIcons registrationId={registration.id} presence={presence} />
         );
       },
+      sorter: (a: Registration, b: Registration) => {
+        const presenceSortingOrder = [
+          'NOT_PRESENT',
+          'LATE',
+          'UNKNOWN',
+          'PRESENT',
+        ];
+        if (
+          presenceSortingOrder.indexOf(a['presence']) >
+          presenceSortingOrder.indexOf(b['presence'])
+        )
+          return 1;
+        else return -1;
```

Comment:
If we are going to shorten it down by skipping curly brackets, might we also throw in a ternary?
```suggestion
        return presenceSortingOrder.indexOf(a['presence']) > presenceSortingOrder.indexOf(b['presence']) ? 1 : -1;
```
or even neater
```suggestion
        return (presenceSortingOrder.indexOf(a['presence']) - presenceSortingOrder.indexOf(b['presence']));
```